scripts/build.js

```diff
@@ -86,13 +86,11 @@ if (POSTINSTALL_BUILD_CWD !== CWD) {
           })
           .value()
           .join(' ');
-
-        Promise.try(function() {
-          if (!_.isEmpty(installArgs)) {
-            console.log('Installing dependencies');
-            return exec('npm install ' + installArgs, opts);
-          }
-        })
+        var needsDepInstallation = !_.isEmpty(installArgs);
```

Comment:
do these need to be var? can they be const?

Reply:
Done!  Was not sure what files are transpiled and which ones weren't, so I adhered to whatever the current file used (`const` if it had `const` or `let`, otherwise `var`)

Reply:
I believe nothing is transpiled anymore, both babel and support for node 6 were dropped.